### ISSUE TITLE
Reference loss charts?

### ISSUE THREAD

**Issue by zaptrem** [NVlabs/DDO#4]
Can you share screenshots of reference loss charts for DDO on VAR, and if possible the shift in cross entropy loss against real data over time as well (even though you don't train on it, it's interesting to watch it change)? I reimplemented it to confirm my understanding and the charts are... interesting.

**Comment by zaptrem** [comment]
Also @zhengkw18 the paper says "For VAR, we observe rapid convergence and only finetune for 2 rounds." Does this mean you tried more rounds and found no improvement?

**Comment by zhengkw18** [comment]
Thanks for your interest in our work.

> Can you share screenshots of reference loss charts for DDO on VAR, and if possible the shift in cross entropy loss against real data over time as well (even though you don't train on it, it's interesting to watch it change)?

Sorry that we no longer maintain the training logs. As long as the FID is decreasing, the training is normal. You can use our training code on VAR as a reference.

> The paper says "For VAR, we observe rapid convergence and only finetune for 2 rounds." Does this mean you tried more rounds and found no improvement?

Yes, even if we sweep the hyperparameters
